modules/mobile/app/controllers/mobile/v0/folders_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require_dependency 'mobile/messaging_controller'
+require_dependency 'mobile/v0/folder_serializer'
+
+module Mobile
+  module V0
+    class FoldersController < MessagingController
+      def index
+        resource = client.get_folders
+        resource = resource.paginate(pagination_params)
```

Comment:
not sure we'll ever hit this endpoint with pagination_params (as they can be blank) but invalid pagination_params seem to throw a 500 rather than a 404 or 422